### ISSUE TITLE
Investigate Historical Notebook-Style Reporters (e.g "Rodney's Notes")

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#518]
So far we've run into the following:

 - Rodney's Notes
 - Ridgely's Notebook
 - Hilles Notebook

It's altogether unclear what we should do with these citations, how common they are, and what they actually are historically. If we're going to start tracking them in CourtListener, we'll need a researcher to actually look into these (and any others that come up).

Looks to be an extreme minority of our opinions though.

**Comment by mlissner** [comment]
Some new ones, with associated counts:

('Ridgely', 286)
('Wilson', 162),
('Clayton', 105),
('Bayard', 100),
('Rodney', 62),
('Wells', 14),
('Miller', 13),
('Read', 11),
('Hilles', 5),
('Rodney Notes', 1),

I think at least some of these are actually in the Indigo Book.

**Comment by mlissner** [comment]
Nope. Didn't see these. This is a task for a librarian or researcher at this point. This should also go under the reporter_db, technically.

**Comment by rlfordon** [comment]
These all look to be old Delaware reporters, from 1792-1830.  Here's the book on [Hathi Trust](http://hdl.handle.net/2027/inu.30000107800843) and on [WorldCat](http://www.worldcat.org/oclc/2882722).  Should I add entries for each of these in the reporter_db?

**Comment by mlissner** [comment]
Nice find on Hathitrust. I got sucked into this last night and did most of the work of putting this into the reporters_db. It's probably good enough for now, though the dates could be improved by going into the book itself and figuring out when each notebook starts and ends, rather than using the broader start and end dates of the entire volume.

But, I think that work is probably better handled in freelawproject/reporters-db#7, which I just created, so I'm going to call this one DONE.

Thanks for your help with this one. Glad to have it taken care of.
